### DISCUSSION TITLE
We are hiring Senior Java Backend Developer (quarkus)

### DISCUSSION THREAD

**Discussion by MargoLHR** [community/community#146038] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

**Description:**

"Burn bright, not out" is the core principle of LEIME’s philosophy. Our corporate culture is built on innovation, empathy, and a healthy work-life balance.

As a company, we help leading brands in iGaming, Crypto, and Tech niches grow in Google SERP and in the eyes of their target audience through PR in top-tier media, expert content, and proprietary SEO strategies.
As a team, we inspire, teach, and support each other. We value not just experience but also soft skills and shared values to ensure we grow together as a united force.

If you’re dreaming of a job that motivates you, a team that supports and empowers your ideas, and career growth without burnout, LEIME is just what the doctor ordered. Join us and grow with us!

**Requirements:**

- Practical experience of 5+ years in web application development, including at least 3 years with Java using Quarkus;
- Proven experience applying cloud platforms in development;
- Hands-on experience with Kubernetes;
- Experience working with Elasticsearch;
- Proficiency with message brokers such as RabbitMQ and Kafka;
- Strong knowledge of relational databases;
- Experience in creating REST APIs;
- Expertise in developing and deploying microservices;
- Deep understanding of the software development lifecycle (SDLC) and modern CI/CD practices.

**Responsibilities:**

- Develop high-availability and scalable backend solutions;
- Select and implement technological solutions aligned with business needs;
- Design and optimize project architecture;
- Maintain and enhance the product’s microservices and APIs;
- Collaborate with frontend developers, QA specialists, and other team members;
- Review work and mentor colleagues as needed.

**Nice-to-have:**

- Experience in full-stack development (React and Next.js);
- Practical experience with Hetzner Cloud;
- Advanced knowledge of Kafka;
- Professional experience with PostgreSQL;
- Experience with Keycloak and Gatekeeper;
- Ability to build backend solutions from scratch;
- Previous experience working in a startup environment.

**Working Conditions:**

- Opportunities for self-realization and career growth as our company evolves rapidly;
- A team of passionate and skilled professionals, making work enjoyable both on and off the clock;
- A great office in Minsk with a hybrid work format / Remote work option available;
- Competitive salary indexed in USD;
- Biannual salary reviews;
- Final salary discussed with the successful candidate. 

![2024-11-25 15 02 55](https://github.com/user-attachments/assets/a594138d-9750-4aee-aa26-c9627c7c24ed)

**Comment by shinybrightstar** [comment]
Hi @MargoLHR , we're happy you're here 😀

However, we currently do not allow job listings on the Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
